schematron/v2.2.0/v2-2-0_L200_Audit.csv

```diff
@@ -323,8 +323,8 @@
 "","","","","","","","","","auc:Quantity","","",""
 "","","","","","","","","","auc:LinkedPremises","","",""
 "","","","","","/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Systems/auc:ConveyanceSystems/auc:ConveyanceSystem/auc:LinkedPremises","false","","","//auc:Buildings/auc:Building[@ID = current()/auc:Building/auc:LinkedBuildingID/@IDref]","auc:ConveyanceSystem must be linked to an auc:Building","",""
-"Historical Energy Use","ASHRAE 211 6.1.2","Monthly Utility Data","ASHRAE 211 6.1.2.1","Monthly Utility Data - Resource Uses","/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Reports/auc:Report/auc:Scenarios/auc:Scenario[auc:ScenarioType/auc:CurrentBuilding/auc:CalculationMethod/auc:Measured]/auc:ResourceUses","true","","","auc:ResourceUse[auc:EnergyResource/text() = 'Electricity']","There must be at least one Electricity ResourceUse","",""
-"","","","","","/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Reports/auc:Report/auc:Scenarios/auc:Scenario[auc:ScenarioType/auc:CurrentBuilding/auc:CalculationMethod/auc:Measured]/auc:ResourceUses/auc:ResourceUse","true","","","auc:EnergyResource","","",""
+"Historical Energy Use","ASHRAE 211 6.1.2","Monthly Utility Data","ASHRAE 211 6.1.2.1","Monthly Utility Data - Resource Uses","/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Reports/auc:Report/auc:Scenarios/auc:Scenario[auc:ScenarioType/auc:CurrentBuilding/auc:CalculationMethod/auc:Measured]/auc:ResourceUses","true","","","auc:ResourceUse[auc:EnergyResource/text() = 'Electricity' and auc:EndUse/text() = 'All end uses' and auc:UtilityIDs/auc:UtilityID]","There must be at least one Electricity ResourceUse","",""
```

Comment:
We only make the more indepth requirements for auc:ResourceUses that are linked to a Utility (ie submeters don't have to provide all of this info)